wikimedia/wikimedia-wordpress-annual-report-plugin
# #988 - Previous Reports Carousel

- Creates a block for the previous reports, which will infinitely scroll.
- Also addresses bugs in other carousels. 

To test:
- Insert the `Report Archive` block (or better still the Previous Reports Pattern)
- Add multiple slides and populate their contents. You will need to add a slide for each previous annual report.
- View the carousel on the front end, confirm that the infinite scroll works as expected, and the carousel will slide with touch on mobile.

Demo:
https://cln.sh/b0Xn9F3t1CzXBHtqlgkd
